apps/xprof_gui/new-priv/src/actions/NavigationActions.js

```diff
@@ -103,7 +107,9 @@ export const queryKeyDown = key => async (dispatch, getState) => {
         const params = `query=${chosenQuery}`;
         const endpoint = `${START_MONITORING_FUNCTION_URL}?${params}`;
         const { error } = await callApi(endpoint);
-        if (error) console.log('ERROR: ', error);
+        if (error) {
+          dispatch(noSuchFunctionError());
+        }
```

Comment:
Pull new version of callApi from `release_2.0`. I changed handling of http request, I don't like `fetch`. Anyway now error have following shape:
```js
error = {
  status: <<http_code>>
  message: <<body_of_the_resposne>>
}